.eslintrc.cjs

```diff
@@ -37,6 +40,28 @@ module.exports = {
         'eslint-comments/disable-enable-pair': ['warn', {
             allowWholeFile: true
         }],
+        'simple-import-sort/imports': ['error', {
+            groups: [
+                // For each group, put type-only imports first.
+                // Side-effect imports
+                ['^\\u0000'],
+                // Node builtin modules
+                [`^(${builtinModulesJoined})(?=/|$).*\\u0000$`, `^(${builtinModulesJoined})(/|$)`],
```

Comment:
Huh, didn't notice that. The `(?=/|$)` is there to ensure we don't match e.g. `fs-utils` as `fs` and therefore a node builtin.

I guess it should be `^(${builtinModulesJoined})(/.*)?\\u0000$` so that we allow e.g. `fs/promises`.